src/interfaces/GeoChart.ts

```diff
@@ -84,6 +91,8 @@ export interface IGeoConfig {
     mapboxToken: string;
     separators?: ISeparators;
     viewport?: IGeoConfigViewport;
+    groupNearbyPoints?: boolean;
+    pushpin?: IGeoPushPinConfig;
```

Comment:
According to the mockup, I suggest to 
- replace `config.pushpin` with `config.points`
- move `groupNearbyPoints` into `config.points`
What do you think?

c.c @tomassatinsky

Reply:
sounds OK to me